src/components/UserProjectsPage/UserProjectsPage.css

```diff
@@ -1,38 +1,30 @@
-
-.ProjectList{
-    display: flex;
-    flex-wrap: wrap;
-    align-items: flex-start;
-    flex-direction: row;
-    width: 100%;
-}
-
-.ProjectList .CardImageDiv{
-    background-size: unset!important;
+.ProjectList {
+  display: grid;
+  gap: 2rem;
+  grid-template-columns: repeat(auto-fill, minmax(15rem, 1fr));
```

Comment:
this is not scalable... it has to keep track of the width of the children... let's say u choose to change the project card width to 10rem, u have to change this rule as well

Reply:
here's my suggestion
[
<img width="261" alt="Screenshot 2020-06-29 at 16 43 07" src="https://user-images.githubusercontent.com/24539185/86013300-ab616880-ba27-11ea-9358-3a45837f16d5.png">
](url)

Reply:
besides, your rule seems complex to comprehend

Reply:
also let the gap be `1rem`

Reply:
Sorry... have just seen this. Let me change the rule and let you know

Reply:
@mrwagaba I have made the requested changes. Please review.